rust/feature-flags/src/flags/test_flag_matching.rs

```diff
@@ -0,0 +1,3721 @@
+#[cfg(test)]
+mod tests {
```

Comment:
This entire file is copied from the tests section of `flag_matching.rs`. No need to review.

Reply:
bless, this will be great for operating in these files